cedar-policy-validator/src/cedar_schema/grammar.lalrpop

```diff
@@ -82,21 +85,35 @@ match {
 
     // other tokens
     ",", ";", ":", "::", "{", "}", "[", "]",
-    "<", ">", "=", "?",
+    "<", ">", "=", "?", "@", "(", ")",
 
 }
 
+#[inline]
+AnyIdent: Node<AnyId> = {
+    <l:@L> <id:IDENTIFIER> <r:@R>
+        => Node::with_source_loc(AnyId::new_unchecked(id), Loc::new(l..r, Arc::clone(src))),
+}
+
+// Annotations := {'@' IDENTIFIER '(' String ')'}
+Annotation: Node<Annotation> = {
+    <l:@L> "@" <key:AnyIdent> <value: ("(" <STR> ")")> <r:@R> => Node::with_source_loc((key, value), Loc::new(l..r, Arc::clone(src)))
```

Comment:
Doesn't look like this handles `@foo` empty annotations. Ideally we could use the same grammar production to ensure they're exactly the same

Reply:
Done.